perfkitbenchmarker/linux_benchmarks/netperf_benchmark.py

```diff
@@ -377,40 +376,28 @@ def RunNetperf(vm, benchmark_name, server_ip, num_streams):
               'sending_thread_count': num_streams,
               'max_iter': FLAGS.netperf_max_iter or 1}
 
-  netperf_cmd = ('{netperf_path} -p {{command_port}} -j {verbosity} '
-                 '-t {benchmark_name} -H {server_ip} -l {length} {confidence}'
-                 ' -- '
-                 '-P ,{{data_port}} '
-                 '-o {output_selector}').format(
-                     netperf_path=netperf.NETPERF_PATH,
-                     benchmark_name=benchmark_name,
-                     server_ip=server_ip,
-                     length=FLAGS.netperf_test_length,
-                     output_selector=OUTPUT_SELECTOR,
-                     confidence=confidence,
-                     verbosity=verbosity)
+  netperf_cmd = (f'{netperf.NETPERF_PATH} -p {{command_port}} -j {verbosity} '
+                 f'-t {benchmark_name} -H {server_ip} -l {FLAGS.netperf_test_length} {confidence}'
+                 f' -- '
+                 f'-P ,{{data_port}} '
+                 f'-o {OUTPUT_SELECTOR}')
 
   if benchmark_name.upper() == 'UDP_STREAM':
-    netperf_cmd += (' -R 1 -m {send_size} -M {send_size} '.format(
-        send_size=FLAGS.netperf_udp_stream_send_size_in_bytes))
-    metadata[
-        'netperf_send_size_in_bytes'] = FLAGS.netperf_udp_stream_send_size_in_bytes
+    send_size = FLAGS.netperf_udp_stream_send_size_in_bytes
+    netperf_cmd += f' -R 1 -m {send_size} -M {send_size} '
+    metadata['netperf_send_size_in_bytes'] = FLAGS.netperf_udp_stream_send_size_in_bytes
 
   elif benchmark_name.upper() == 'TCP_STREAM':
-    netperf_cmd += (' -m {send_size} -M {send_size} '.format(
-        send_size=FLAGS.netperf_tcp_stream_send_size_in_bytes))
-    metadata[
-        'netperf_send_size_in_bytes'] = FLAGS.netperf_tcp_stream_send_size_in_bytes
+    send_size = FLAGS.netperf_tcp_stream_send_size_in_bytes
+    netperf_cmd += f' -m {send_size} -M {send_size} '
+    metadata['netperf_send_size_in_bytes'] = FLAGS.netperf_tcp_stream_send_size_in_bytes
 
   if FLAGS.netperf_thinktime != 0:
-    netperf_cmd += (' -X {thinktime},{thinktime_array_size},'
-                    '{thinktime_run_length} ').format(
-                        thinktime=FLAGS.netperf_thinktime,
-                        thinktime_array_size=FLAGS.netperf_thinktime_array_size,
-                        thinktime_run_length=FLAGS.netperf_thinktime_run_length)
+    netperf_cmd += (f' -X {FLAGS.netperf_thinktime},{FLAGS.netperf_thinktime_array_size},'
```

Comment:
If the first line has to be split I tihnk this is easier to read:
(' -X '
 f'{FLAGS.netperf_thinktime},'
 f'{FLAGS.netperf_thinktime_array_size},'
 f'{FLAGS.netperf_thinktime_run_length} ')

But ugly anyway it goes